.gitignore

```diff
@@ -1,31 +1,13 @@
 ## FOLDERS
 /.sass-cache/
 /node_modules
-vendor/bin
-vendor/dealerdirect
-vendor/mediawiki/mediawiki-codesniffer
-vendor/phpcompatibility
-vendor/php-parallel-lint
-vendor/sirbrillig
-vendor/squizlabs
-vendor/wp-coding-standards
-vendor/automattic/jetpack-autoloader
-/wpcom-test-backup/
+/vendor/
 /.vscode/
 /logs
 /allure-results/
 
 /projects/plugins/**/node_modules/
```

Comment:
This should probably get moved too.

Also the `tests/php/files/jetpack-150x150.jpg` and `/projects/plugins/jetpack/modules/**/*.min.css.map` below, which GitHub won't let me comment on (grumble grumble).

Reply:
I left it in since I would imagine there wouldn't be a case where we would want `node_modules` committed. But probably could use `node_modules/` for that instead of the plugin-specific one.

What do you think?

Reply:
I look at it the same as vendor. I can't think why we'd ever want it in the monorepo. Most plugins will probably just copy a `/node_modules` rule into their plugin-level .gitignore, but if one wants to do something weird like Jetpack does with vendor it can do it there.

Reply:
👍 I can jive with that.

Reply:
db0b9483e -- just removed it from the repo-level one. It was already present in the Jetpack plugin one.